cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/query/IndexManagerTest.java

```diff
@@ -30,84 +32,99 @@
 public class IndexManagerTest extends AbstractIndexTestBase {
 
     @Test
-    public void enusureIndexedGeneratesIndexName() {
+    public void enusureIndexedGeneratesIndexName() throws QueryException {
         assertThat(im.ensureIndexed(Arrays.<FieldSort>asList(new FieldSort("name"))), is(notNullValue()));
     }
 
     @Test
-    public void deleteFailOnNoIndexName() {
+    public void deleteFailOnNoIndexName() throws QueryException {
```

Comment:
This test can't be split as it tests a certain sequence of events - create index, delete non-existent index, check that first index is still there.